### DISCUSSION TITLE
PiStorm32 with Indivision Mk3

### DISCUSSION THREAD

**Discussion by Exc3ss** [PiStorm/pistorm32-lite-hardware#3] [Q&A]
I know the PISTORM emulates an RTG board, I own an Indivision MK3 which allows remapping the A1200 video output to HDMI or VGA. If I insert a PISTORM32 on my A1200 and instead of using the PISTORM's HDMI output, I wanted to use the HDMI output of the Indivision, would it work? And if I then wanted to switch from a program that supports RTG video at 32-bit resolution, which the PISTORM provides through RTG emulation, to a game, would I still see everything from the Indivision's output?

**Comment by spacemishka** [comment]
It does work.... have seen it on a popular Youtube-Channel....Video was released around Feb '23
